test/prototype_transforms_kernel_infos.py

```diff
@@ -61,11 +61,10 @@ def __init__(
         self.reference_inputs_fn = reference_inputs_fn
 
 
-DEFAULT_IMAGE_CLOSENESS_KWARGS = dict(
-    atol=1e-5,
-    rtol=0,
-    agg_method="mean",
-)
+DEFAULT_PIL_REFERENCE_CLOSENESS_KWARGS = {
+    (("TestKernels", "test_against_reference"), torch.float32, "cpu"): dict(atol=1e-5, rtol=0, agg_method="mean"),
+    (("TestKernels", "test_against_reference"), torch.uint8, "cpu"): dict(atol=1e-5, rtol=0, agg_method="mean"),
+}
```

Comment:
Two lessons here:

1. We only need the tolerances for the reference tests, but they were applied everywhere. There was no harm here, but this could have hidden bugs before.
2. Some of our kernels like `resize` produce quite large differences for some pixels, which are hidden by aggregating them. Maybe we should review and use stricter "default" tolerances and just increase it for the few that need more.